source/slang/slang-reflection-api.cpp

```diff
@@ -1888,7 +1894,19 @@ namespace Slang
                     bindingRange.descriptorRangeCount++;
                 }
 
+                auto bindingRangeIndex = m_extendedInfo->m_bindingRanges.getCount();
+
                 m_extendedInfo->m_bindingRanges.add(bindingRange);
+
+                // For `StructuredBuffer` fields, we also make sure to report it as a sub-object range.
+                if (auto structuredBufferTypeLayout = as<StructuredBufferTypeLayout>(typeLayout))
+                {
+                    TypeLayout::ExtendedInfo::SubObjectRangeInfo subObjectRange;
+                    subObjectRange.bindingRangeIndex = bindingRangeIndex;
+                    subObjectRange.offsetVarLayout = createOffsetVarLayout(typeLayout, path);
+                    subObjectRange.spaceOffset = 0;
```

Comment:
Is a `spaceOffset` of zero necessarily correct here?

Reply:
I think this is just a copy paste from existing code, so I am not sure if it is needed or correct...